modules/Bio/EnsEMBL/Variation/Pipeline/VariationConsequence_conf.pm

```diff
@@ -263,7 +263,7 @@ sub pipeline_analyses {
           },
           { -logic_name => 'by_gene_transcript_effect',
             -module => 'Bio::EnsEMBL::Variation::Pipeline::TranscriptEffect',
-            -hive_capacity  => 150,
+            -hive_capacity  => 50,
```

Comment:
If the hive_capcity is being reduced to 50, should the analysis_capacity also be reduced from 60 to 50?